lib/openstudio-standards/standards/ashrae_90_1/ashrae_90_1_2004/data/ashrae_90_1_2004.spc_typ.json

```diff
@@ -18549,10 +18549,10 @@
       "occupancy_activity_schedule": "SchoolSecondary ACTIVITY_SCH",
       "is_residential": null,
       "ventilation_standard": "ASHRAE 62.1-1999",
-      "ventilation_primary_space_type": "Education",
-      "ventilation_secondary_space_type": "Corridors",
-      "ventilation_per_area": 0.1,
-      "ventilation_per_person": 0.0,
+      "ventilation_primary_space_type": "Offices",
+      "ventilation_secondary_space_type": "Reception Areas",
+      "ventilation_per_area": null,
+      "ventilation_per_person": 15.0,
```

Comment:
The ventilation primary and secondary space types were changed based on 62.1 and the corresponding ventilation rates were updated.